cookbook/case_studies/ml_training/nlp_processing/word2vec_and_lda.py

```diff
@@ -285,21 +281,22 @@ def word_movers_distance(model_ser: FlyteFile[MODELSER_NLP]) -> float:
 # =====================================
 #
 # The word embeddings made by the model can be visualized after reducing the dimensionality to two with t-SNE.
-@task(cache_version="1.0", cache=True, limits=Resources(mem="600Mi"))
+@task(cache_version="1.0", cache=True, limits=Resources(mem="1000Mi"))
```

Comment:
had to bump this as suggested by @eapolinario  was getting similar error (memory related) when executing task in cluster as mentioned here https://flyte-org.slack.com/archives/C043P7NDKL6/p1666911022385569
Not sure if there is any upper bound which it needs to be kept under ?

Reply:
For the demo cluster, 1000Mi is the max. You can, however, configure it.